### DISCUSSION TITLE
abicio or abjicio?

### DISCUSSION THREAD

**Discussion by fiapps** [DivinumOfficium/divinum-officium#3075] [Developers corner]
I was going to open an issue to complain that 54f8927 had changed _abiciate_ to _abjiciate_ in the antiphon for the _Benedictus_ on the feast of St. Cecilia, but then I discovered usage is divided about 50-50 in the repo. Should we be preferring one of these two forms? I thought we were preferring the former, but now I'm not so sure.

The question also regards other words derived from _iacio._ _Abicio_ is considered more classical, so it's what the 1961 _editio typica_ used, but it's also likely to be mispronounced, because people who haven't studied Latin won't know that the single i [should be pronounced as if it were two](https://www.perseus.tufts.edu/hopper/text?doc=Perseus%3Atext%3A1999.04.0001%3Asmythp%3D6). Thus former breviary editors, who were more concerned with helping readers than being classical, wrote _abjicio._

**Comment by APMarcello3** [comment]
Closed by ccd42d8ab01eeeb43707fad4bbf367d4c9ab0651.

**Reply by fiapps** [reply]
OK, so implicitly the answer is abicio, but there are still a lot of uses of abjicio, conjicio, dejicio, injicio, … left in the repo.

**Reply by APMarcello3** [reply]
Are they appearing in the Rubrics 1960 version? If so, please provide a list?

**Reply by fiapps** [reply]
I find them by searching the data files (try the regular expression `\b(ab|con|de|in)j[ií]ci`), so it would be easier to provide a PR changing them than a list of items to change. They may not all appear with the 1960 rubrics, but those using other rubrics will see inconsistent spelling if we don't follow the same convention in all texts. If you want to use a different convention with different rubrics, you should have the orthography adjusted by the code when producing output, like is done for j to i conversion. `spell_var` could split the non-HTML `@parts` into words, and replace words that occur in a dict mapping one style of spelling to the other.
